### PR TITLE
update downloads manager with new commands

### PR DESCRIPTION
**Pull request by ron-myers**
## Description

- adds `Copy Latest Download` command
- adds `Open Downloads Folder` command

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/downloads-manager/src/open-downloads.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { open, popToRoot, showHUD } from "@raycast/api";
+import { runAppleScript } from "run-applescript";
+
+const openFinderToDownloadsAppleScript = `do shell script "open ~/Downloads/"`;
+
+export default async function main() {
+  await runAppleScript(openFinderToDownloadsAppleScript);
```
Comment (thomaspaulmann) [inline extensions/downloads-manager/src/open-downloads.tsx:7]: No need for AppleScript in this case. just use our `open` or `showInFinder` function.

```suggestion
  const downloadsdir = join(homedir, "Downloads");
  await showInFinder(downloadss);
```

You need to fix the imports ;-)

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Thanks for the additional commands, they are great! Just left a comment to improve it a bit with some built-in functions. Otherwise it's ready to ship!

File: extensions/downloads-manager/src/open-downloads.tsx (hunk 1 above)
Comment (ron-myers) [reply extensions/downloads-manager/src/open-downloads.tsx:7]: Cool!  I will switch to using that approach.

**Comment by ron-myers** [comment]
There was a merge issue after for #3148 - hopefully has all of the changes as you saw them @thomaspaulmann 

I added infused-kim as a contributor but the username doesn't match

`Invalid contributor "infused-kim". Use the Raycast username`

So then removed the line in `package.json`.

**Comment by infused-kim** [comment]
> There was a merge issue after for #3148 - hopefully has all of the changes as you saw them @thomaspaulmann
> 
> I added infused-kim as a contributor but the username doesn't match
> 
> `Invalid contributor "infused-kim". Use the Raycast username`
> 
> So then removed the line in `package.json`.

Thank you! My raycast user name is `Kimcha`. Perhaps it would be better if you added it in this PR as mine was already merged and if I push the change it will create another merge conflict.

What do you think?

**Comment by ron-myers** [comment]
Sure thing.  Adding now.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**